.github/workflows/coverity.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  schedule:
+    - cron:  '0 4 * * 6'
+
+env:
+  CI_MAKE: make -f .travis.mk
+
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    container:
+      image: ghcr.io/avtikhon/tarantool/testing/debian-buster:latest
```

Comment:
It would be nice to have some sort of README about the pipeline and especially about this docker image (how and from which source it was built) because it would be devastating to try to rebuild the image without that info (in case of losing the original image or needing to update some soft inside it)

Reply:
We already have comments on it in .travis.mk file which has special targets to create/check these images.

Reply:
Looked through the .travis.mk, did not find any references to build information for the image. If you think README is an overkill, some comment in yaml would be sufficient I guess. I just don't want CI moving to a pitfall where it can't be recreated from scratch and only works in some "cloud" depending on some external registries

Reply:
Added comments to images where to find instructions how recreate it.